lib/utilities/lang.js

```diff
@@ -2,7 +2,9 @@
 // Thanks Yahoo!
 // http://yuilibrary.com/yui/docs/api/files/yui_js_yui-lang.js.html
 
-var TOSTRING     = Object.prototype.toString;
+/*eslint-disable */
```

Comment:
changing other people's* ? That makes sense. Maybe files like this should be in a different directory.